### DISCUSSION TITLE
Table: Column filters sort value options

### DISCUSSION THREAD

**Discussion by reuvygroovy** [grafana/grafana#38124] [Dashboards, Panels & Visualizations]
When filtering by values, shouldn't you sort the numbers in order?
![image](https://user-images.githubusercontent.com/1325158/124106875-55296a00-da6d-11eb-8fc0-404d003c05ae.png)

We are using Grafana 7.5.1

**Comment by torkelo** [comment]
For numerical values that could make sense, for alphabetical not sure.

**Comment by reuvygroovy** [comment]
For numeric, theoretically you should sort also, and/or be able to identify that the number is numeric and sort accordingly.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
